src/test/skript/tests/regressions/pull-6746-spawn-dropped-item.sk

```diff
@@ -0,0 +1,10 @@
+test "spawn dropped item":
+	spawn dropped stone at spawn of world "world":
+		set {_e} to entity
+	assert {_e} is set with "failed to spawn dropped stone"
+	delete entity within {_e}
+
+
+	spawn dropped item at spawn of world "world":
+		set {_e2} to entity
+	assert {_e2} is not set with "spawned `dropped item` despite undefined item type"
```

Comment:
```suggestion
	assert {_e2} is not set with "spawned `dropped item` despite undefined item type"

```

Reply:
why? There's already a newline at the end of the file.

Reply:
<img width="1023" alt="image" src="https://github.com/SkriptLang/Skript/assets/14147477/c6fbbd36-9766-4feb-88fd-3b565085d9e1">
Nothing shows up in the file explorer, are you sure? It says it's a 10-line file and that's the 10th line 😕

Reply:
It would show the little red do not enter sign

Reply:
This symbol may only appear on files with extensions recognised by Github, which is useful as these are often standards for the languages in question. I guess that this symbol does not appear here, even though an end line is missing

Reply:
no, it does show up for .sk files. the simple answer is that a new line is not missing! It just doesn't show the next line.

it's `abc\n` which displays the same as `abc` when nothing follows it.

![image](https://github.com/SkriptLang/Skript/assets/10354869/0be714f6-fd89-47d0-bd61-3277ad60ab75)

Reply:
Okay 😄